test/prism/errors_test.rb

```diff
@@ -1871,6 +1871,16 @@ def test_upcase_end_in_def
       ]
     end
 
+    def test_warnings_verbosity
+      warning = Prism.parse("def foo; END { }; end").warnings[0]
+      assert_equal "END in method; use at_exit", warning.message
+      assert_equal 1, warning.level
```

Comment:
Do we want fancier values here for the level or integers are fine?
If yes, which Symbol should they be for both warnings levels and the error level?
(in Java they could be an `enum`, basically a copy of the C enum)

Reply:
I switched to Symbols, it seems much better compatibility-wise.